setup.cfg

```diff
@@ -41,5 +41,5 @@ galaxy_importer =
     ansible_test/build_template.yaml
 
 [coverage:report]
-fail_under = 93.7
+fail_under = 94.2
```

Comment:
Unrelated change?

Reply:
I usually bump this up in same PR if coverage increases, keeps it from creeping back down